builtin/game/register.lua

```diff
@@ -155,9 +192,28 @@ function core.register_item(name, itemdef)
 	getmetatable(itemdef).__newindex = {}
 
 	--core.log("Registering item: " .. itemdef.name)
-	core.registered_items[itemdef.name] = itemdef
-	core.registered_aliases[itemdef.name] = nil
+	core.registered_items[name] = itemdef
+	core.registered_aliases[name] = nil
 	register_item_raw(itemdef)
+
+	if itemdef.can_place then
+		core.override_on_place(name, function(itemstack, placer, pt)
```

Comment:
So you're overriding `on_place` when there is a `can_place`, but you don't call any preexisting `on_place`?